### PR TITLE
PAY-658: Payout report view

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) adds a Payout reports topic to my Magento Payments integration branch for review by internal devs.

Jira: https://jira.corp.magento.com/browse/PAY-658

Preview build: [#22](https://docs.magedevteam.com/22/user-guide/magento-payments/payouts.html)

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- /user-guide/magento-payments/payoutreports.html

### CODE REVIEW

File: src/magento-payments/payoutreports.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+By default, the Payout reports view shows the last seven days of daily reports. You can also [create custom reports](#create-a-new-payout-report) based on a specified timeframe.
+
+To view a report:
+
+1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payout reports**.
+1. Click a payout report from the reports list.
+1. View [information for each transaction](#column-descriptions) in the daily report, including transaction date, reference ID, invoice number, and payment method details.
+
+## Create a new Payout report
```
Comment (imeron2433) [inline src/magento-payments/payoutreports.md:41]: The word "Create" makes me feel weird here. Maybe something like "Customize"?

File: src/magento-payments/payoutreports.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+## Download Payout reports
+
+1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payout reports**.
+1. To download multiple reports, check to select them from the list and click the _Download_ (![]({% link images/images/btn-download.png %})) icon.
+1. To download a single report, click the _Download_ (![]({% link images/images/btn-download.png %})) icon in the Actions column for a specific report.
+
+Your report(s) are downloaded in a .csv format.
+
+## Available Payout reports
```
Comment (imeron2433) [inline src/magento-payments/payoutreports.md:57]: I would change this to something like "Default" Views or something

**Review by imeron2433 — COMMENTED** [review]
Just a few comments. Looks good!

File: src/magento-payments/payoutreports.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+Reference ID | Original PayPal transaction ID for which this event is related
+Invoice | Invoice ID (one per order) of the transaction
+Magento order | Magento order ID
+Magento trans | Magento transaction ID
+Pay method | Credit card type (BANK, PAYPAL, CREDIT_CARD) and associated card provider (Visa, MasterCard, etc.)
+Trans amt | Amount of the transaction
+Cur (trans amt) | Currency unit for transaction amount
+Pending | Amount yet to be disbursed
+Cur (pending) | Currency unit for the pending amount
+Seller amt | Amount of funds transferred to or from a customer; Funds moving out of the seller account show a dash (-) prefix.
```
Comment (leslietilling) [inline src/magento-payments/payoutreports.md:93]: We might want to discuss this use of `;` in meeting. It hasn't been done in User Guide, but we can talk more about solutions for mixing non-sentence forms with sentences in tables.

File: src/magento-payments/payoutreports.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+Cur (recoup amt) | Currency unit for the recoup amount
+{:style="table-layout:auto;"}
+
+## Transaction types
+
+These transaction types may be noted in Payout reports.
+
+Report | Description
+------------ | --------------------
+PAYMENT | Money moved between a buyer and a seller for an order
```
Comment (leslietilling) [inline src/magento-payments/payoutreports.md:115]: The All Caps thing is another question. Even when used in the UI, we have been steering away from it in docs.  But I am thinking you might have these as placeholders for now :)

**Review by leslietilling — APPROVED** [review]
Just some basic comments and suggested edits

File: src/magento-payments/payoutreports.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
+---
+ee_only: false
+title: Payout Reports
+---
+
+Payout reports show comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+You can efficiently cross-reference orders and payments and reconcile accounts---all from one place.
```
Comment (rancollier) [inline src/magento-payments/payoutreports.md:7]: The user can efficiently cross-reference because the payments and reconcile accounts are in the same place. Would it be helpful to explain what makes it efficient?

Comment (shrielenee) [reply src/magento-payments/payoutreports.md:7]: Yes, especially given our audience. Good reminder.

File: src/magento-payments/payoutreports.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payout reports in the Admin.
+
+You can download payout reports in a .csv file format for use in existing accounting or order management software.
+
+## Availability
+
+On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payout reports** to see Payout reports.
```
[the remark this answers is not in the document]
Comment (shrielenee) [reply src/magento-payments/payoutreports.md:15]: In the mockup, Magento Payments is actually clickable. That's why it's bolded. I will update further down the line if that changes.

File: src/magento-payments/payoutreports.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+- `View` - Use this function to view the details of the report.
+- `Download` - Use this function to save the report as an .cvs file.
+
+## View Payout reports
+
+By default, the Payout reports view shows the last seven days of daily reports. You can also [create custom reports](#create-a-new-payout-report) based on a specified timeframe.
+
+To view a report:
+
+1. On the _Admin_ sidebar, go to **Sales** > **Magento Payments** > **Payout reports**.
```
[the remark this answers is not in the document]
Comment (shrielenee) [reply src/magento-payments/payoutreports.md:37]: I'm not sure what you mean here. Are you suggesting I remove "On the Admin sidebar," or remove the whole step?

[the remark this answers is not in the document]
Comment (leslietilling) [reply src/magento-payments/payoutreports.md:37]: I meant the same comment for that menu command formatting. But you responded that the middle item is selectable, so all good.

File: src/magento-payments/payoutreports.md (hunk 3 above)
Comment (shrielenee) [reply src/magento-payments/payoutreports.md:93]: What is your suggestion in lieu of this? None of the other rows have complete sentences so this was my way of conforming to that convention while also adding the secondary info.

File: src/magento-payments/payoutreports.md (hunk 4 above)
Comment (shrielenee) [reply src/magento-payments/payoutreports.md:115]: I am representing it as it looks in the UI, correct. Given these are statuses, and those are typically pulled from code, I formatted them as per how they view for the user to keep things clear.

**Review by rancollier — APPROVED** [review]

File: src/magento-payments/payoutreports.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+ee_only: false
+title: Payout Reports
+---
+
+Payout reports show comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+You can efficiently cross-reference orders and payments and reconcile accounts---all from one place.
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payout reports in the Admin.
+
+You can download payout reports in a .csv file format for use in existing accounting or order management software.
```
Comment (rancollier) [inline src/magento-payments/payoutreports.md:11]: What are your thoughts on changing "download" for "export"?

File: src/magento-payments/payoutreports.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
+---
+ee_only: false
+title: Payout Reports
+---
+
+Payout reports show comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+You can efficiently cross-reference orders and payments and reconcile accounts---all from one place.
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payout reports in the Admin.
```
Comment (rancollier) [inline src/magento-payments/payoutreports.md:9]: Thoughts on "see" for "view"?

Comment (shrielenee) [reply src/magento-payments/payoutreports.md:10]: View is mentioned above. Leaving this.

File: src/magento-payments/payoutreports.md (hunk 8 above)
Comment (shrielenee) [reply src/magento-payments/payoutreports.md:12]: The mechanism is known as a download mechanism, and the button graphic is indicative of that and conventionally familiar as such. I'm inclined to leave this.

**Review by imeron2433 — APPROVED** [review]

File: src/magento-payments/payouts.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
+---
+ee_only: false
+title: Payouts
+---
+
+Payouts shows comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+
+You don't have to open multiple dashboards or views to cross-reference orders and payments or reconcile accounts. Magento Payments enables you to take all these actions from one place---Payouts---so that you can view and manage your payouts efficiently.
```
Comment (rancollier) [inline src/magento-payments/payouts.md:8]: Having these views consolidated in one view can allow the user to be more efficient at managing payout but effective at understanding what is going on. 

This is just a thought: 

Orders, payments and reconciliations are represented in a single view for efficient cross referencing from one place---Paymouts---. This enables you to be conversant and manage payouts.

**Review by rancollier — APPROVED** [review]
It looks good.

File: src/magento-payments/payouts.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+See payments across multiple providers, as well as the linked Magento order and transaction IDs, transaction amounts, payment method per transaction, and more, all within Payouts in the Admin.
```
Comment (h-madan) [inline src/magento-payments/payouts.md:10]: We're not onboarding any other provider except PayPal for GA.
This mention about multiple providers may create further questions.
We should get someone from the Product team to review this please.

Comment (shrielenee) [reply src/magento-payments/payouts.md:10]: Good point @h-madan

**Comment by cruxburn** [comment]
Couple questions/comments:
- Is this working framework for the Reporting documentation with content based on current know detail and content will be updated once MVP version of Payout report are finalized and completed?
- In bullet 2 of Download transactions section the HTML internal link label discrepancy from "specify your preferred date range." and the link tile of "Customize transactions timeframe" feel a bit clunky. Suggest using the same words/language in either the link or the linked to give visual continuity
- In the opening paragraph, first sentence I would change "payment balance" to "payment amount" and make that the first item in the list before processed volume, as this is the most import item in a payout report, the actual payment.

**Comment by shrielenee** [comment]
> Couple questions/comments:
> 
> * Is this working framework for the Reporting documentation with content based on current know detail and content will be updated once MVP version of Payout report are finalized and completed?
> * In bullet 2 of Download transactions section the HTML internal link label discrepancy from "specify your preferred date range." and the link tile of "Customize transactions timeframe" feel a bit clunky. Suggest using the same words/language in either the link or the linked to give visual continuity
> * In the opening paragraph, first sentence I would change "payment balance" to "payment amount" and make that the first item in the list before processed volume, as this is the most import item in a payout report, the actual payment.

* This content is a baseline for the Payouts documentation. The content will be revised closer to MVP if/when things change in the UI, functionality, etc. Pre-writing documentation is an effort to have something ready, that can be revised later if need-be, and is accomplished WHILE the feature is being developed so that my work is paired with dev work (which guarantees better accuracy and comprehensiveness).
* Ok... the end user only sees the link text though, "specify your preferred date range", so I'm not sure consistency is needed but I will reword this to be a little more parallel---"customize the date range timeframe for your transactions".
* Done.

**Review by cruxburn — APPROVED** [review]
Format and updates approved from PM
